src/endpoints/workerskv/mod.rs

```diff
@@ -30,9 +32,25 @@ impl ApiResult for Vec<WorkersKvNamespace> {}
 #[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq)]
 pub struct Key {
     pub name: String,
+    #[serde(default)]
+    #[serde(deserialize_with = "deserialize_option_timestamp")]
     pub expiration: Option<DateTime<Utc>>,
 }
 
+pub fn deserialize_option_timestamp<'de, D>(
+    deserializer: D,
+) -> Result<Option<DateTime<Utc>>, D::Error>
```

Comment:
I wonder if this return type should be `Result<DateTime<Utc>>, D::Error>` instead. You could argue that this function should only return `Ok` if the JSON response is deserialized into an actual DateTime<Utc>. 

From the perspective of the application code using `cloudflare-rs`, if I call an endpoint that's supposed to contain a timestamp, and it doesn't return an actual timestamp, it's a `Err` not `Ok`. So maybe instead of returning `Ok(None)` it should return `Err`. What do you think, as the person who's actually using this endpoint?

Reply:
Awesome PR!

If I understand him correctly, I agree with @adamchalmers. It feels like the deserializer should be agnostic of whether the value it's deserializing is wrapped in an optional. I was therefore expecting more of a `deserialize_timestamp` rather than a `deserialize_option_timestamp`. Am I missing a nuance here?

Reply:
Since `expiration` an optional value, I think it still makes sense to return an Option? Not having `None` would make the semantics of using the `Key` struct intuitive, and also would not play well with the serde optional deserialization logic we're using.

Reply:
Oh, I didn't realize that `expiration` is defined as optional in the API. Nevermind then, I agree with your implementation.